index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Youtube video player</title>
+    <!-- adding css styles -->
+    <style>
+        body {
+            background-color: black;
+            color: rgb(231, 217, 217);
+            text-align: center;
+        }
+    
+        header,
+        main,
+        aside {
+            padding: 20px;
+            margin: 10px auto;
+            max-width: 800px;
+            background-color: #070707;
+            border-radius: 10px;
+        }
+    
+        iframe {
+            display: block;
+            margin: 0 auto;
+        }
+    
+        h2 {
+            font-size: 24px;
+            font-weight: bold;
+            margin-bottom: 10px;
+        }
+    
+        a {
+            color: red;
+            text-decoration: none;
+            display: block;
+            margin-bottom: 5px;
+        }
+    
+        a:hover {
+            color: #050505;
+            background-color: rgb(6, 6, 7);
```

Comment:
this style tag should be in a different CSS folder. For the code to look cleaner